### PR TITLE
0.9.6 update

### PR DESCRIPTION
**Pull request by jj16791**
This PR updates various values and documentation to reflect the 0.9.6 version release

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
-Welcome to the 0.9.5 open source release of the Simulation Engine! SimEng is a fast, easy to use and modify, cycle-level simulator for CPUs. Its initial focus is on simulating single Arm cores in server CPUs, and so the instruction set architecture (ISA) target is initially Arm9.2-a+SVE+SME. RISC-V has also seen some inital support, starting with rv64ima. Later versions of the Arm ISA, additional RISC-V ISA extensions and other ISAs, will be supported in future releases.
+Welcome to the 0.9.6 open source release of the Simulation Engine! SimEng is a fast, easy to use and modify, cycle-level simulator for CPUs. Its initial focus is on simulating single Arm cores in server CPUs, and so the instruction set architecture (ISA) target is initially Arm9.2-a+SVE+SME. RISC-V has also seen some inital support, starting with rv64ima. Later versions of the Arm ISA, additional RISC-V ISA extensions and other ISAs, will be supported in future releases.
```
Comment (dANW34V3R) [inline RELEASE-NOTES.txt:1]: This line needs updating RE RISC-V. Currently supports rv64imafd and c hopefully

Comment (jj16791) [reply RELEASE-NOTES.txt]: Changed to rv64imafd for now

Comment (dANW34V3R) [reply RELEASE-NOTES.txt]: I'd maybe remove "initial support" now as well as this is a typical set of extensions and it is a bit more filled out than before. ~"There is also RISC-V support covering rv64imafd"

Comment (jj16791) [reply RELEASE-NOTES.txt]: Will make that change

**Review by dANW34V3R — APPROVED** [review]

File: docs/sphinx/index.rst [hunk 2]
```diff
@@ -44,13 +44,13 @@ SimEng is a framework for building modern, cycle-accurate processor simulators.
...
+Currently, SimEng targets the Armv9.2-a ISA with support for the SVE, SVE2, and SME extensions as well as RISC-V rv64imafd. SimEng has the ability to model up to out-of-order, superscalar, single-core processors, and to emulate a subset of Linux system-calls. It supports statically compiled C and Fortran binaries that run on real hardware, with additional support for single-threaded OpenMP binaries too. Internally, SimEng currently models memory as an infinite L1 cache, i.e. it assumes that all loads and stores hit the L1 cache. However, we have a tested integration with the `Structural Simulation Toolkit <http://sst-simulator.org/>`_ (SST) allowing for a full memory model to be simulated; more information can be found in the :doc:`SST Integration section <sst/index>`.
```
Comment (JosephMoore25) [inline docs/sphinx/index.rst:53]: `rv64iamfdc` - we missed the `c` off.

File: docs/sphinx/index.rst [hunk 3]
```diff
@@ -70,15 +70,17 @@ SimEng was first presented by `Professor Simon McIntosh-Smith <http://uob-hpc.gi
...
+This is SimEng's seventh release, and should be considered beta level software (version 0.9.6). We expect you to find issues, primarily in unimplemented instructions or unimplemented system calls. Please let us know when you hit these, either by submitting a pull request (PR), or by filing an issue on the Github repo. You can find the all the code and associated test suites for SimEng in the `GitHub repository <https://github.com/UoB-HPC/SimEng>`_. The file `RELEASE_NOTES.txt <https://github.com/UoB-HPC/SimEng/blob/main/RELEASE-NOTES.txt>`_, found in the root of the project, explains the status of the project and includes other relevant information from the SimEng development team.
```
Comment (JosephMoore25) [inline docs/sphinx/index.rst:83]: Eighth release now? I'm assuming we want to increment vs the previous release

**Review by JosephMoore25 — CHANGES_REQUESTED** [review]

File: RELEASE-NOTES.txt [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 
 You can find all the user and developer documentation on the SimEng webpage:
 
    https://uob-hpc.github.io/SimEng/index.html
 
-SimEng should be fast, much faster than other cycle-level CPU simulators. The fastest hardware we’ve run it on is an M1-Ultra Mac Studio, where we’ve measured SimEng running at up to ~1 MIPS for the Out of Order (OoO) core in our ThunderX2 and A64fx models. SVE (vector) codes are more complex to simulate and tend to be slower - on the same hardware we saw simulated SVE codes running at around 0.6 MIPS.
+SimEng should be fast, much faster than other cycle-level CPU simulators. The fastest hardware we’ve run it on is an M1-Ultra Mac Studio, where we’ve measured SimEng running at up to ~1.2 MIPS for the Out of Order (OoO) core in our ThunderX2 and A64fx models. SVE (vector) and SME (matrix) codes are more complex to simulate and tend to be slower - on the same hardware we saw simulated SVE codes running at around 0.9 MIPS and SME codes at around 0.8 MIPS.
```
[the remark this answers is not in the document]
Comment (jj16791) [reply RELEASE-NOTES.txt:9]: Did averages not peak, my bad

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (jj16791) [reply RELEASE-NOTES.txt]: updated to include the c extension where appropriate

File: docs/sphinx/index.rst (hunk 3 above)
Comment (jj16791) [reply docs/sphinx/index.rst:83]: Good spot

File: docs/sphinx/index.rst (hunk 2 above)
Comment (jj16791) [reply docs/sphinx/index.rst]: Duplicate comment

**Review by FinnWilkinson — APPROVED** [review]

**Review by dANW34V3R — APPROVED** [review]

File: docs/sphinx/index.rst (hunk 2 above)
Comment (JosephMoore25) [reply docs/sphinx/index.rst]: Didn't notice, cheers.

**Review by JosephMoore25 — APPROVED** [review]

File: docs/sphinx/index.rst [hunk 5]
```diff
@@ -70,15 +70,17 @@ SimEng was first presented by `Professor Simon McIntosh-Smith <http://uob-hpc.gi
 
 Additionally, other works concerning SimEng and its use can be found below:
 
+- Second International workshop on RISC-V for HPC - `An Empirical Comparison of the RISC-V and AArch64 Instruction Sets <https://dl.acm.org/doi/abs/10.1145/3624062.3624233>`_
```
Comment (ABenC377) [inline docs/sphinx/index.rst:73]: Can we add something to this to make the year clearly available.  Maybe "at SC2023"?  I think its good to immediately see that this work is current

File: docs/sphinx/index.rst [hunk 6]
```diff
@@ -70,15 +70,17 @@ SimEng was first presented by `Professor Simon McIntosh-Smith <http://uob-hpc.gi
...
+This is SimEng's eighth release, and should be considered beta level software (version 0.9.6). We expect you to find issues, primarily in unimplemented instructions or unimplemented system calls. Please let us know when you hit these, either by submitting a pull request (PR), or by filing an issue on the Github repo. You can find the all the code and associated test suites for SimEng in the `GitHub repository <https://github.com/UoB-HPC/SimEng>`_. The file `RELEASE_NOTES.txt <https://github.com/UoB-HPC/SimEng/blob/main/RELEASE-NOTES.txt>`_, found in the root of the project, explains the status of the project and includes other relevant information from the SimEng development team.
```
Comment (ABenC377) [inline docs/sphinx/index.rst:83]: typo: "the all the code"

**Review by ABenC377 — CHANGES_REQUESTED** [review]
Just a couple of comments about index.rst

File: docs/sphinx/index.rst (hunk 5 above)
Comment (jj16791) [reply docs/sphinx/index.rst:73]: Sounds good

File: docs/sphinx/index.rst (hunk 6 above)
Comment (jj16791) [reply docs/sphinx/index.rst:83]: Good spot

**Review by JosephMoore25 — DISMISSED** [review]

**Review by ABenC377 — DISMISSED** [review]

**Review by FinnWilkinson — APPROVED** [review]

**Review by dANW34V3R — APPROVED** [review]

**Review by FinnWilkinson — APPROVED** [review]

**Review by ABenC377 — APPROVED** [review]

**Review by dANW34V3R — APPROVED** [review]

**Pull request merged**